### ISSUE TITLE
Jest issues 

### ISSUE THREAD

**Issue by KyleGashler** [JacobWeisenburger/zod_utilz#11]
Having a hard time tracking down the change that triggered this because your release notes aren't very detailed but it looks like you introduced some strange module export patterns or something along those lines. Could you just look through this error output and point me in the direction of something you changed since 0.7.3 that might have triggered it? Our jest transformers already account for typescript and tsx so i'm thinking it has something to do with your module pattern. 

```
Jest encountered an unexpected token

Jest failed to parse a file. This happens e.g. when your code or its dependencies use non-standard JavaScript syntax, or when Jest is not configured to support such syntax.
```

**Comment by JacobWeisenburger** [comment]
Can you give me more info about your dev environment?
- OS
- Runtime
   - node
   - deno
   - bun
- jest version

The more info you can give me, the easier it will be to fix this.

**Comment by KyleGashler** [comment]
- OS: Mac Sanoma 14
- Runtime: Node 20.11.1
- Jest version: "jest": "29.7.0"

I'm not sure what else to provide you. When upgrading from 0.7.3 to your latest version I started seeing these errors.

**Comment by JacobWeisenburger** [comment]
It seems this issue is caused by the fact that I'm using bun now to build. But bun build doesn't support cjs yet. https://bun.sh/docs/bundler#format. So until they do, you will have to use 0.7.3. Is that ok with you?

**Comment by KyleGashler** [comment]
absolutely. Just wanted to see the path forward. I appreciate you looking into it.

**Comment by JacobWeisenburger** [comment]
Great, thanks
